tests/acceptance/ManageSubscriptionLinkCest.php

```diff
@@ -62,12 +62,42 @@ public function manageSubscriptionLink(\AcceptanceTester $i) {
     $i->seeNoJSErrors();
   }
 
-  public function unsubscribeLink(\AcceptanceTester $i) {
-    $i->wantTo('Verify that "unsubscribe" link works and subscriber status is set to unsubscribed');
-    $this->sendEmail($i);
+  public function unsubscribeLinksWithLinkTracking(\AcceptanceTester $i) {
```

Comment:
This makes me think - do we still track unsubscribe as a "click" and do we still want to?

Reply:
Very good question. We track it. I haven't thought about removing the tracking. It makes sense to remove it or maybe at least filter these clicks from stats because unsubscribes are not kind of clicks user want to get. Not sure if removing it has some other consequences.